### PR TITLE
Feature/add timer

### PR DESCRIPTION
**Pull request by yiranli624**
## Is this a fix or a feature?
Feature.

## What is the change?
A timer feature will appear when "START ACTIVITY" button is clicked. the new activity section will also be hidden from display.

## What does it fix? 
N/A.

## Where should the reviewer start?
Index.html : line 13;
main.js : line 10, 74;
styles.css : line 145.

## How should this be tested?
By clicking on start activity button.

### Screenshot (optional)

@sarah01123
@elle624
@atownse

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -7,39 +7,42 @@ var exerciseButton = document.querySelector('.exercise');
 var accomplishInput = document.querySelector('.accomplish-input');
 var minutesInput = document.querySelector('.minutes-input');
 var secondsInput = document.querySelector('.seconds-input');
-
+var buttonText = document.querySelectorAll('.cat-titles')
```
Comment (saraho1123) [inline main.js:10]: Good idea using a query ALL here!

File: main.js [hunk 2]
```diff
@@ -7,39 +7,42 @@ var exerciseButton = document.querySelector('.exercise');
...
 startActivityBtn.addEventListener('click', createCurrentActivity);
 
-function changeColor(button, category) {
-    button.classList.add(category);
-}
-
-function returnDefaultColor(button1, category1, button2, category2) {
-    button1.classList.remove(category1);
-    button2.classList.remove(category2);
+function changeColor(button1, category1, button2, category2, button3, category3) {
```
Comment (atownse) [inline main.js:23]: Great job getting this to work, I see we needed an option for if only one button is defined

**Review by atownse — COMMENTED** [review]
Good work getting the page to shift correctly

**Review by saraho1123 — APPROVED** [review]
Looks like everything is working like it is supposed to. Brilliant move with that query ALL for the category buttons!

**Pull request merged**